src/styles/index.scss

```diff
@@ -1,3 +1,261 @@
+
+    &-item.is-active::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 4px;
+      background-color: #00acdc;
+      border-radius: 8px;
+      transition: width 0.3s ease;
+    }
+
+    &-item:hover::after {
+      width: 100%;
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+// Card
+.card {
+  box-sizing: border-box;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  height: 408px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  position: relative;
+  transition: transform 0.3s ease;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-main-color;
+    margin-bottom: 4px;
+    transition: color 0.3s ease;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-secondary-color;
+    line-height: 14px;
+    text-align: left;
+    position: relative;
+  }
+
+  &__stars-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-secondary-color;
+    text-align: left;
+    position: relative;
+  }
+
+  &__total {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $text-main-color;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  &__button {
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    right: 16px;
+    background: $color-blue-accent;
+    border: 1px solid $color-blue-accent;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $color-white;
+    box-sizing: border-box;
+  }
+
+  &__button:hover {
+    color: $color-blue-accent;
+    background-color: $color-white;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__name {
+      color: $blue-color-hover;
+    }
+  }
+}
+
+p {
+  margin: 0;
+}
+
+.card-stars {
+  display: flex;
+  gap: 4px;
+  margin-bottom: 24px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url('../images/star-active.svg');
+  }
+
+  &__star:nth-child(5) {
+    background-image: url('../images/star.svg');
+  }
+}
+
+// Card__container
+.cardСontainer {
```

Comment:
The class name `cardСontainer` uses a non-standard character (Cyrillic 'С'). It's recommended to use standard ASCII characters to avoid potential issues.